### PR TITLE
IAP-4630: change redis documentation ...

### PR DESCRIPTION
**Pull request by kvedenskii-axway**
* remove redis installation instructions
* add link to redis documentation

Thank you for your contribution to the Axway-Open-Docs repo.

## Describe the changes

Enter a brief description of your changes to communicate to the reviewers what you changed and why.

## Deploy preview link

Please add the deploy preview link to the **specific page** that you've changed.

_Although Netlify will add this link below, it will redirect to the main page of the documentation, not to an specific page._

## Checklist for contributors

Before submitting this PR, please make sure:

* [ ] You have read the [contribution guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/)
* [ ] You have signed the [Axway CLA](https://cla.axway.com/)
* [ ] You have verified the technical accuracy of your change
* [ ] You have verified that your change does not expose any sensitive information (passwords, keys, etc.)
* [ ] You have followed the [Markdown guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/writing_markdown/)  (unless this is is a Netlify CMS contribution)
* [ ] You have verified that all status checks have passed

_Put an x in the boxes that apply. This is simply a reminder of what we are going to look for before merging your change._

### CODE REVIEW

**Review by jowens-axway — APPROVED** [review]

File: content/en/docs/apim_installation/apiportal_install/install_software_redis.md [hunk 1]
```diff
@@ -1,99 +1,77 @@
...
 
-   ```shell
-   sudo subscription-manager repos --enable rhel-server-rhscl-7-rpms
-   sudo subscription-manager repos --enable rhel-7-server-optional-rpms
-   sudo yum clean all
-   ```
+For better performance and scalability, you can configure API Portal to cache APIs in a Redis cache. Using a Redis cache is recommended if you plan to expose hundreds of APIs, or you plan to connect API Portal to more than one API Manager.
 
-2. Install Redis:
+## Prerequisites
```
Comment (andreamussap) [inline content/en/docs/apim_installation/apiportal_install/install_software_redis.md:13]: Have you all looked at the deploy preview of this page? This Heading 2 doesn't make sense. I believe it was forgotten here !?

File: content/en/docs/apim_installation/apiportal_install/install_software_redis.md [hunk 2]
```diff
@@ -1,99 +1,77 @@
...
   "date": "2019-08-09",
-  "description": "Optionally install a Redis cache for better performance."
+  "description": "Optionally configure a Redis cache for better performance."
 }
-For better performance and scalability, you can configure API Portal to cache APIs in a Redis cache. Using a Redis cache is recommended if you plan to expose hundreds of APIs, or you plan to connect API Portal to more than one API Manager.
-
-## Install Redis on Red Hat 7 (RHEL 7) using RHSCL
 
-1. Enable `rhel-server-rhscl-7-rpms` and `rhel-7-server-optional-rpms` repositories from RHSCL:
+{{< alert title="Note" color="primary" >}}Please have in mind that from version 2021 August API Portal supports [integration with Elasticsearch](/docs/apim_installation/apiportal_install/install_software_elastic), which allows you to achieve the same caching capabilities and in addition pagination with much better performance. While Redis support only caches API Catalog page, Elasticsearch also supports Applications page caching. We recommend using Elasticsearch instead of Redis. Continue reading this section if you still prefer to use Redis instead of Elasticsearch.{{< /alert >}}
```
Comment (andreamussap) [inline content/en/docs/apim_installation/apiportal_install/install_software_redis.md:9]: FYI - I've left comment about this in the Jira IAP-4630.

File: content/en/docs/apim_installation/apiportal_install/install_software_redis.md [hunk 3]
```diff
@@ -1,99 +1,77 @@
...
-2. Install Redis:
+## Prerequisites
 
-   ```shell
-   sudo yum install rh-redis5
-   ```
+### Install Redis server
 
-3. Enable Redis executables in any bash session by default:
+Obtain a Redis 5 or 6 server from the [official site](https://redis.io/download).
```
Comment (andreamussap) [inline content/en/docs/apim_installation/apiportal_install/install_software_redis.md:17]: When you say '5 or 6' without more context you create uncertainty. 
As a customer, how do I know which one to use? 
Unless there's a reason to choose one or another, please choose an option. e.g., 'Download a Redis 6 from [redis.io](https://redis.io/download)'

File: content/en/docs/apim_relnotes/20210830_apip_relnotes.md [hunk 4]
```diff
@@ -29,6 +29,8 @@ The API and Applications catalogs now support pagination controls to allow a bet
 
 {{< alert title="Note" color="primary" >}}Pagination is only available for software installation of API Portal. This is not available for API Portals deployed in containers.{{< /alert >}}
 
+{{< alert title="Note" color="primary" >}}Integration with Elasticsearch allows you to achieve the same caching capabilities as caching with Redis and in addition pagination with much better performance. While Redis support only caches API Catalog page, Elasticsearch also supports Applications page caching. We recommend using Elasticsearch instead of Redis.{{< /alert >}}
```
Comment (andreamussap) [inline content/en/docs/apim_relnotes/20210830_apip_relnotes.md:32]: @dginevaAxway Have you reviewed this? This is a late update, why are you creating it in the November branch? This piece should be release straight away

Comment (dginevaAxway) [reply content/en/docs/apim_relnotes/20210830_apip_relnotes.md:32]: Actually this needs to go to master and not for November.

**Pull request closed without merging**